geometries/quaternions/rotation_compare.py

```diff
@@ -1,30 +1,79 @@
+"""Usage:
```

Comment:
This script now actually compares the quaternions directly from two or more CKs directly, instead of having separate scripts to write quaternions to two different files.

This new script proved that the two Aspera CKs, ck/cdr_2_ck.bc and ck/aspera_test1.bc, in the test kernel set were identical in S/C attitude over time.  As a result, aspera_test1.bc was removed from the test kernel set.